Sources/Widgets/Widgets3D/AngleWidget/behavior.js

```diff
@@ -75,6 +75,13 @@ export default function widgetBehavior(publicAPI, model) {
   // --------------------------------------------------------------------------
 
   publicAPI.handleMouseMove = (callData) => {
+    if (
+      model.hasFocus &&
+      model.widgetState.getHandleList().length === MAX_POINTS
+    ) {
+      publicAPI.loseFocus();
```

Comment:
I find it odd to loose focus when moving the mouse.
Can you explain why it makes sense to loose focus at that time ?

Reply:
I admit it was mainly to look like the DistanceWidget, which worked like that for 3 years.
But thinking about it, I would find it better to place this code at the beginning of the `handleLeftButtonRelease` method (so we place point + invoke startInteractionEvent during press, and lose focus + invoke endInteractionEvent during release).  WDYT?

Reply:
ok to have it in handleLeftButtonRelease. 

I do not have an opinion wether it is at the beginning or at the end.